frontend/packages/dev-console/src/components/taskruns/list-page/TaskRunsListPage.tsx

```diff
@@ -14,17 +15,19 @@ const TaskRunsListPage: React.FC<Omit<
   React.ComponentProps<typeof ListPage>,
   'canCreate' | 'kind' | 'ListComponent' | 'rowFilters'
 > &
-  TaskRunsListPageProps> = (props) => {
+  TaskRunsListPageProps> = ({ hideBadge, ...props }) => {
+  const searchParams = getURLSearchParams();
+  const kind = searchParams?.kind;
+
   return (
     <ListPage
       {...props}
-      canCreate={false}
+      canCreate={kind?.includes(referenceForModel(TaskRunModel)) ?? false}
```

Comment:
Curious, why this change?

Reply:
this adds the `create task run` button in the search page when the Task Runs resource type is selected